### PR TITLE
Add docs for `LoRA` pass

### PR DESCRIPTION
**Pull request by jambayk**
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

File: docs/source/features/passes/pytorch.md [hunk 1]
```diff
@@ -2,6 +2,30 @@
 
 PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
 
+## LoRA
```
[the remark this answers is not in the document]
Comment (jambayk) [reply docs/source/features/passes/pytorch.md]: Resolved

File: docs/source/features/passes/pytorch.md [hunk 2]
```diff
@@ -2,6 +2,30 @@
 
 PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
 
+## LoRA 
+Low-Rank Adaptation, or `LoRA`, is a fine-tuning approach which freezes the pre-trained model weights and injects trainable rank decomposition matrices (called adapters) into the layers of the model.
```
[the remark this answers is not in the document]
Comment (jambayk) [reply docs/source/features/passes/pytorch.md]: Resolved

File: docs/source/features/passes/pytorch.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jambayk) [reply docs/source/features/passes/pytorch.md]: Resolved

File: docs/source/features/passes/pytorch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jambayk) [reply docs/source/features/passes/pytorch.md]: Resolved

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**